src/main/java/org/jenkinsci/plugins/mesos/MesosAgentSpecTemplate.java

```diff
@@ -78,6 +81,7 @@ public MesosAgentSpecTemplate(
     this.jvmArgs = jvmArgs;
     this.additionalURIs = additionalURIs;
     this.nodeProperties = nodeProperties;
+    this.containerImage = StringUtils.isNotBlank(containerImage) ? containerImage : null;
```

Comment:
Why not use `Optional.OfNullable` here instead?

Reply:
changed to move optional here